src/test/java/com/google/sps/ImageSelectionTest.java

```diff
@@ -0,0 +1,81 @@
+package com.google.sps.image;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.io.IOException;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class ImageSelectionTest {
+
+  private Set<String> keywords;
+
+  private static final String bingImgLinkStart = "https://tse";
+
+  private static void checkLink(ImageSelection imageSelection) {
```

Comment:
The checkLink name does not tell me what it asserts.

For example `assertTrue(boolean)` is a good name because it tells the reader that we want to assert that the argument is true.

However, in your case, checkLink doesn't tell me what it asserts.